endpoint/endpoint.go

```diff
@@ -26,6 +26,14 @@ const (
 	OwnerLabelKey = "owner"
 )
 
+// TTL is a structure defining the TTL of a DNS record
+type TTL struct {
+	// The value of the TTL
+	Value int64
+	// Whether or not the TTL value is configured
+	IsConfigured bool
```

Comment:
Why do you need this attribute?
I guess you don't need this struct and only need a TTL as int64.